### PR TITLE
fix handle l4 proxy err

### PR DESCRIPTION
**Pull request by Poorunga**
fix handle l4 proxy err : lb: service xxx has no instances, refer to issue #74

### CODE REVIEW

File: agent/pkg/chassis/registry/registry.go [hunk 1]
```diff
@@ -96,14 +96,16 @@ func (esd *EdgeServiceDiscovery) FindMicroServiceInstances(consumerID, microServ
 	}
 
 	// gen MicroServiceInstances
+	// TODO(Poorunga) needs optimization
```
Comment (khalid-jobs) [inline agent/pkg/chassis/registry/registry.go:99]: If you need a todo here, you should make it clear, what do you want to  optimizaton? Or we just delete the comment.

Comment (Poorunga) [reply agent/pkg/chassis/registry/registry.go:99]: OK, delete it.

**Review by fisherxu — COMMENTED** [review]
The commit message should describe the commit, instead of write the issue number.

**Comment by Poorunga** [comment]
> The commit message should describe the commit, instead of write the issue number.

Done.  @fisherxu

**Pull request merged**